### ISSUE TITLE
Re:clarifying README for Windows install

### ISSUE THREAD

**Issue by CometS1** [vshelke/bookpro#36]
I would like to have the instructions developing in docker clarified for how to develop. Please add what docker is and a link to its download to the README, as well as issues that may occur when attempting to develop in a windows environment. When attempting to setup my environment to develop using docker, I got the error that my source path did not exist when attempting to use the [docker run] command.

**Comment by vshelke** [comment]
Ok sure @CometS1 thank you for pointing this issue out. I shall update the README accordingly. 👍

**Comment by vshelke** [comment]
Hi @CometS1 can you test the current docker instruction and inform if everything is working fine ?

**Comment by CometS1** [comment]
Currently I receive an error when running [docker-compose up]  that says "/usr/bin/env: python\r: No such file or directory" and it fails.

![image](https://user-images.githubusercontent.com/25065296/47370607-1f286b00-d6b4-11e8-8d8d-3f0fa22af499.png)

**Comment by CometS1** [comment]
After some checking online, it seems that this error occurs because the manage.py file format is set for Macs, which is why is says python\r, as the \r is specific for Macs. I solved the problem by opening the manage.py file in vi and entering `:set ff=unix` and then `:wq`. It may be possible to also solve this problem by entering `dos2unix manage`, though I haven't tested this. I found the solution :[here](https://stackoverflow.com/questions/19425857/env-python-r-no-such-file-or-directory).
